packages/opentelemetry-instrumentation-openai/tests/traces/test_prompt_caching.py

```
@@ -67,13 +67,12 @@ def test_openai_prompt_caching(instrument_legacy, span_exporter, log_exporter):
     assert cache_creation_span.attributes["gen_ai.completion.0.role"] == "assistant"
     assert cache_read_span.attributes["gen_ai.completion.0.role"] == "assistant"
 
-    assert cache_creation_span.attributes["gen_ai.usage.prompt_tokens"] == 1149
-    assert cache_creation_span.attributes["gen_ai.usage.completion_tokens"] == 315
-    assert cache_creation_span.attributes["gen_ai.usage.cache_read_input_tokens"] == 0
+    assert cache_creation_span.attributes["gen_ai.usage.input_tokens"] == 1149
+    assert cache_creation_span.attributes["gen_ai.usage.output_tokens"] == 315
+    assert cache_creation_span.attributes["gen_ai.usage.gen_ai.usage.cache_read_input_tokens"] == 0
```

Ruff (0.12.2)

Use of `assert` detected

(S101)